expat/CMakeLists.txt

```diff
@@ -395,6 +395,13 @@ if(EXPAT_SHARED_LIBS)
     if(MSVC)
         set(expat_SRCS ${expat_SRCS} lib/libexpat.def)
     endif()
+    if(WIN32)
+        # Add DLL version
+        string(REPLACE "." "," DLL_VERSION ${CMAKE_PROJECT_VERSION})
```

Comment:
Two notes here;
- Would you mind using `_EXPAT_DLL_VERSION` for the variable name? That would communicate it being internal and being about Expat.
- `CMAKE_PROJECT_VERSION` seems to  be the topmost project version. I think we need `PROJECT_VERSION` here, in case Expat used as a sub project of something.
  - https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION.html
  - https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_VERSION.html